crates/plugins/src/manifest.rs

```diff
@@ -30,6 +30,9 @@ impl PluginManifest {
     pub fn name(&self) -> String {
         self.name.to_lowercase()
     }
+    pub fn license(&self) -> String {
```

Comment:
Our usual convention for functions like this (and I can't remember where it came from) has been to skip the clone and return `&str`, as often callers don't need to own a copy, and if they do they can do the clone.  In this case, of course, it makes no odds, and we're certainly not consistent about doing it...!

Reply:
Definitely an unnecessary clone. Thanks for pointing this out @itowlson